src/main/java/cz/smarteon/loxone/app/state/SwitchState.java

```diff
@@ -0,0 +1,40 @@
+package cz.smarteon.loxone.app.state;
+
+/**
+ * Enum containing the different switch states for a <code>SwitchControl</code>.
+ */
+public enum SwitchState {
```

Comment:
well the ON/OFF/UNITIALIZED semantics is basicallyl everywhere in Loxone - not only tied to SwitchControl - so let's make it more generic. Also may be simple `Boolean - true/false/null` would fit as well?